docs/DeveloperGuide.md

```diff
@@ -86,6 +86,55 @@ Guarantees: Editing of internship application is successful
   1b1. InternHunter displays an error message and informs the user of the valid input format. </br>
   Use case resumes from step 1.
 
+### Use case: UC10 - Delete a skill or achievement
+
+#### MSS
+
+Precondition: User already has an existing list of skills and achievements. </br>
+Guarantees: Deletion of skill or achievement is successful
+
+1.  User requests to delete a skill or achievement from the user profile and provides the index.
+2.  InternHunter removes the skill or achievement from the user profile. </br>
+    Use case ends.
+
+#### Extensions
+
+1a. InternHunter detects that the index entered is out of bounds. </br>
+  1a1. InternHunter displays an error message and informs the user of the valid input format. </br>
+  Use case resumes from step 1.
+
+### Use case: UC11 - Edit a skill or achievement
+
+#### MSS
+
+Precondition: User already has an existing list of skills and achievements. </br>
+Guarantees: Editing of skill or achievement is successful
+
+1.  User requests to edit the details of a skill or achievement and inputs the index and details.
+2.  InternHunter edits the skill or achievement from the user profile. </br>
+    Use case ends.
+
+#### Extensions
+1a. InternHunter detects that the index entered is out of bounds. </br>
+  1a1. InternHunter displays an error message and informs the user of the valid input format. </br>
+  Use case resumes from step 1.
+  
+1b. InternHunter detects an error in the input format. </br>
+  1b1. InternHunter displays an error message and informs the user of the valid input format. </br>
+  Use case resumes from step 1.
+
```

Comment:
I am unsure about this myself but what if there are no items in the list? what kind of interactions can the user and system have? I know your preconditions addresses that but do we have to consider the case where you call edit and there are no items in the list?

Reply:
perhaps once we merge the similar use cases together, we can include more extensions to it!